docs/quickstarts/ruby-quickstart.md

```diff
@@ -0,0 +1,187 @@
+stubs from [helloworld.proto](https://github.com/grpc/grpc/tree/{{site.data.config.grpc_release_branch}}/examples/protos/helloworld.proto),
+but you'll need the tools for the rest of our quickstart, as well as later
+tutorials and your own projects.
+
+To install gRPC tools, run:
+
+```sh
+gem install grpc-tools
+```
+
+## Download the example
+
+You'll need a local copy of the example code to work through this quickstart. Download the example code from our Github repository (the following command clones the entire repository, but you just need the examples for this quickstart and other tutorials):
+
+```sh
+  $ # Clone the repository to get the example code:
+  $ git clone https://github.com/grpc/grpc
+  $ # Navigate to the "hello, world" Ruby example:
+  $ cd grpc/examples/ruby
+```
+
+## Run a gRPC application
+
+From the `examples/python` directory:
+
+1. Run the server
+
+   ```sh
+   $ ruby greeter_server.rb
+   ```
+
+2. In another terminal, run the client
+
+   ```sh
+   $ ruby greeter_client.rb
+   ```
+
+Congratulations! You've just run a client-server application with gRPC.
+
+## Update a gRPC service
+
+Now let's look at how to update the application with an extra method on the
+server for the client to call. Our gRPC service is defined using protocol
+buffers; you can find out lots more about how to define a service in a `.proto`
+file in [What is gRPC?]() and [gRPC Basics: Ruby][]. For now all you need
+to know is that both the server and the client "stub" have a `SayHello` RPC
+method that takes a `HelloRequest` parameter from the client and returns a
+`HelloResponse` from the server, and that this method is defined like this:
+
+
+```proto
+// The greeting service definition.
+service Greeter {
+  // Sends a greeting
+  rpc SayHello (HelloRequest) returns (HelloReply) {}
+}
+
+// The request message containing the user's name.
+message HelloRequest {
+  string name = 1;
+}
+
+// The response message containing the greetings
+message HelloReply {
+  string message = 1;
+}
+```
+
+Let's update this so that the `Greeter` service has two methods. Edit `examples/proto/helloworld.proto` and update it with a new `SayHelloAgain` method, with the same request and response types:
+
+```proto
+// The greeting service definition.
+service Greeter {
+  // Sends a greeting
+  rpc SayHello (HelloRequest) returns (HelloReply) {}
+  // Sends another greeting
+  rpc SayHelloAgain (HelloRequest) returns (HelloReply) {}
+}
+
+// The request message containing the user's name.
+message HelloRequest {
+  string name = 1;
+}
+
+// The response message containing the greetings
+message HelloReply {
+  string message = 1;
+}
+```
+
+(Don't forget to save the file!)
+
+## Generate gRPC code
+
+Next we need to update the gRPC code used by our application to use the new service definition. From the `examples/ruby/` directory:
+
+```sh
+$ grpc_tools_ruby_protoc -I ../protos --ruby_out=lib --grpc_out=lib ../protos/helloworld.proto
```

Comment:
This does not work yet but ask discussed with @murgatroid99 it will after we have packages that incorporate grpc/grpc#7551
Currently, the plugin has to be explicitly pointed to.